frontend/src/bundles/common/components/video-modal/libs/components/video-modal-content/components/video-preview/video-preview.tsx

```diff
@@ -0,0 +1,78 @@
+import { Button, Flex, Icon, Text } from '@chakra-ui/react';
+import { faPlay } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useCallback, useState } from 'react';
+
+import {
+    VideoPreview as VideoPreviewValues,
+    VideoSizeLabel,
+} from './libs/enums/enums.js';
+import { type VideoPreview as VideoPreviewType } from './libs/types/types.js';
+
+const VideoPreview = (): JSX.Element => {
+    const [view, setView] = useState<VideoPreviewType>(
+        VideoPreviewValues.PORTRAIT,
+    );
+
+    const setPortraitView = useCallback((): void => {
```

Comment:
```suggestion
    const handleSetPortraitView = useCallback((): void => {
```

A10 https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md